ProiectPractica/Validator/ConfigurationValidator.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Eventing.Reader;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProiectPractica.Validator
+{
+    public class ConfigurationValidator
+    {
+        /// <summary>
+        /// Validates a string so that it can be parsed into an integer.
+        /// Returns true if the string can be parsed as an integer,
+        /// and returns false otherwise.
+        /// </summary>
+        /// <param name="textToBeValidated"></param>
+        /// <param name="errorMessage"></param>
+        /// <returns></returns>
+        public bool ValidateInt(string textToBeValidated, out string errorMessage)
+        {
+            if (textToBeValidated.Length == 0)
+            {
+                errorMessage = "Field should not be empty!";
+                return false;
+            }
+            if (int.TryParse(textToBeValidated, out int integer))
+            {
+                if (integer < 0)
+                {
+                    errorMessage = "The number should be positive!";
+                    return false;
+                }
+                errorMessage = "";
+                return true;
+            }
+            else if ((!textToBeValidated.Substring(1).All(char.IsDigit) && textToBeValidated[0] != '-') || !textToBeValidated.All(char.IsDigit))
+            {
+                errorMessage = "This should be an integer!";
+                return false;
+            }
+            else if (textToBeValidated.Substring(1).All(char.IsDigit) && textToBeValidated[0] == '-' && textToBeValidated.Length > 1)
+            {
+                errorMessage = "The number should be positive!";
+                return false;
+            }
+            errorMessage = "The number is too big!";
+            return false;
+        }
+
+        /// <summary>
+        /// Validates a string so that it can be parsed into a float.
+        /// Returns true if the string can be parsed as a float,
+        /// and it returns false otherwise.
+        /// </summary>
+        /// <param name="textToBeValidated"></param>
+        /// <param name="errorMessage"></param>
+        /// <returns></returns>
+        public bool ValidateFloat(string textToBeValidated, out string errorMessage)
+        {
+            if (textToBeValidated.Length == 0)
+            {
+                errorMessage = "Field should not be empty!";
+                return false;
+            }
+            if(float.TryParse(textToBeValidated, CultureInfo.CreateSpecificCulture("de-DE"), out float number))
```

Comment:
Switch this if to do an early false return, in order to reduce the indentation level